### DISCUSSION TITLE
Jinja2, ricerca di esempi e di documentazione

### DISCUSSION THREAD

**Discussion by ganzerlimino** [indomus/forum#1420] [HUB: Home Assistant]
Ciao a tutti,
volevo iniziare a giocare un pochino con i modelli che si trovano sotto "Strumenti per sviluppatori" per capire meglio come è strutturato il tutto.

Al posto degli esempi, però, vorrei poter usare i dati dei miei sensori come valori di input.

Sicuramente per mio limite, non trovo documentazione su come tirarmi dentro un mio sensore, per ipotesi uno che mi eroga il messaggio MQTT "tasmota/tele/sw_controsoffitto/STATE" , in uno dei modelli di test.

Potreste puntarmi verso qualche pagina che mi dia un minimo di illuminazione su come utilizzare i miei dati in queste strutture di test?

**Comment by snakuzzo** [comment]
Se hai il sensore e devi usare i template, di certo non ragionerai in base ai topic e payload mqtt.
Per i template è scritto tutto in documentazione. Basta cercare e leggere

https://www.home-assistant.io/docs/configuration/templating/

**Reply by ganzerlimino** [reply]
Ciao diego, perfetto, mille grazie della spiegazione, mi metto a spulciare quanto mi hai linkato!
Mino

**Reply by ocimin** [reply]
Ti aggiungo questi riferimenti su Jinja che mi aveva dato tempo fa snakuzzo e che a me erano stati molto utili:

https://jinja.palletsprojects.com/en/3.0.x/

Saluti.

Ocimin

**Reply by ganzerlimino** [reply]
Wow, mille grazie Ocimin, aggiungo agli elementi da studiare!
